src/mlpack/prereqs.hpp

```diff
@@ -34,6 +27,8 @@
 #include <tuple>
 #include <utility>
 
+#include <armadillo>
```

Comment:
That means we should remove this from here for now too.

Reply:
Ok perfect.